uaclient/contract.py

```diff
@@ -68,30 +70,41 @@ def request_root_macaroon(self):
         self.cfg.write_cache('root-macaroon', root_macaroon)
         return root_macaroon
 
-    def request_accounts(self, macaroon_token):
+    def request_accounts(self, macaroon_token=None, machine_token=None):
         """Request list of accounts this user has access to."""
         headers = self.headers()
-        headers.update({'Authorization': 'Macaroon %s' % macaroon_token})
+        if macaroon_token:
+            headers.update({'Authorization': 'Macaroon %s' % macaroon_token})
+        else:
+            headers.update({'Authorization': 'Bearer %s' % machine_token})
         accounts, _headers = self.request_url(
             API_V1_ACCOUNTS, headers=headers)
         self.cfg.write_cache('accounts', accounts)
         return accounts
 
-    def request_account_contracts(self, macaroon_token, account_id):
+    def request_account_contracts(
+            self, account_id, macaroon_token=None, machine_token=None):
         """Request a list of contracts authorized for account_id."""
         url = API_V1_TMPL_ACCOUNT_CONTRACTS.format(account=account_id)
         headers = self.headers()
-        headers.update({'Authorization': 'Macaroon %s' % macaroon_token})
+        if macaroon_token:
+            headers.update({'Authorization': 'Macaroon %s' % macaroon_token})
+        else:
+            headers.update({'Authorization': 'Bearer %s' % machine_token})
```

Comment:
Same issue as above.

Reply:
+ 1 will resolve these as ua-contracts won't support both types of auth, dropping all related

Reply:
really dropped